data/json/monsters.json

```diff
@@ -5963,6 +5963,37 @@
     "flags":["SEES", "HEARS", "SMELLS", "WARM", "BASHES", "GROUP_BASH", "POISON", "BLEED", "CBM_OP", "ACIDPROOF", "NO_BREATHE", "REVIVES", "BONES"]
     },{
     "type" : "MONSTER",
+    "id" : "mon_zombie_armored",
+    "name": "decayed paladin",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"Z",
+    "color":"dkgray_red",
+    "size":"MEDIUM",
+    "material":"steel",
+    "diff":25,
+    "aggression":100,
+    "morale":100,
+    "speed":70,
```

Comment:
Pretty confident that the depowered armor slows PCs who wear it (leg encumbrance) so it should also slow zeds.